packages/manager/apps/communication/public/translations/common/Messages_fr_FR.json

```
@@ -1,6 +1,7 @@
 {
   "title": "Mes communications",
-  "description": "Retrouvez l'ensemble des communications qu'OVH vous a envoyé, exceptées les demandes d'assistance.",
+  "description": "Retrouvez l'ensemble des communications qu'OVH vous a envoyé, exceptées les demandes d'assistance. Vous pouvez les consulter via la section: <anchor>Assistance</anchor>",
+  "assistance_link_label": "Assistance",
```

You could have achieved the same result with this (avoiding to have the "assistance_link_label"):
```suggestion
  "description": "Retrouvez l'ensemble des communications qu'OVH vous a envoyé, exceptées les demandes d'assistance. Vous pouvez les consulter via la section: <anchor label=\"Assistance\"></anchor>",
```